gdpr/en/what-policies.md

```diff
@@ -24,7 +24,7 @@ You may also want to add a consent statement to your Author Submission Checklist
 
 ### OJS/OMP 3
 
-These applications do not currently provide a means to add a consent checkbox item to user registration, but this [has been filed in GitHub](https://github.com/pkp/pkp-lib/issues/3575)  and will be available in OJS/OMP 3.1.1-1, on or before May 25. 
+These applications currently provide a means to add a consent checkbox item to user registration by clicking a checkbox  "Yes, I agree to have my data collected and stored according to the privacy statement." 
 
 You may also want to add a consent statement to your Author Submission Checklist, in Settings > Workflow > Submission > Submission Preparation Checklist.
```

Comment:
This guidance is probably not necessary either. When submitting authors are asked to consent separately from the checklist.

Reply:
yes

Reply:
@withanage sorry maybe there was some confusion. The first paragraph, with your changes, about the checkbox should be kept. But the second paragraph about adding a consent statement to the Author submission Checklist, should be removed.